src/js/tracing/stalltracking.ts

```diff
@@ -215,6 +226,21 @@ export class StallTrackingInstrumentation {
     );
   }
 
+  /**
+   * Switch that enables the iteraction once app moves from background to foreground.
+   */
+  private _backgroundEventListener(state: AppStateStatus): void {
+    if (state === ('active' as AppStateStatus)) {
+      this._isBackground = false;
+      if (this._timeout != null) {
+        this._lastIntervalMs = timestampInSeconds() * 1000;
+        this._iteration();
+      }
+    } else {
+      this._isBackground = true;
```

Comment:
Yes, but I would prefer using the below code since clearTimeout is a void function
```suggestion
      this._isBackground = true;
      clearTimeout(this._timeout);
```